data_processing/common/DataStore.py

```diff
@@ -206,24 +206,24 @@ def setContainer(self, container_id):
             return self
         
         # Set match clause to id
-        self.logger.debug ("Match on: %s", self._container_id)
+        self.logger.debug ("Match on: %s", self._datastore_id)
 
         # Attach
         cohort = Node("cohort", self._namespace_id)
-        if not len(self._conn.query(f""" MATCH (co:{cohort.get_match_str()}) MATCH (container) WHERE id(container) = {self._container_id} MERGE (co)-[:INCLUDE]->(container) RETURN co,container """ ))==1: 
-            self.logger.warning ( "Cannot attach, tried [%s]", f""" MATCH (co:{cohort.get_match_str()}) MATCH (container) WHERE id(container) = {self._container_id} MERGE (co)-[:INCLUDE]->(container) RETURN co,container """)
+        if not len(self._conn.query(f""" MATCH (co:{cohort.get_match_str()}) MATCH (container) WHERE id(container) = {self._datastore_id} MERGE (co)-[:INCLUDE]->(container) RETURN co,container """)) == 1:
+            self.logger.warning ( "Cannot attach, tried [%s]", f""" MATCH (co:{cohort.get_match_str()}) MATCH (container) WHERE id(container) = {self._datastore_id} MERGE (co)-[:INCLUDE]->(container) RETURN co,container """)
             return self
 
         # Let us know attaching was a success! :)
-        self.logger = init_logger(f'logs/container-{self.address}.log')
```

Comment:
I've been meaning to raise this issue, to improve logging-- perhaps it's time to fetch the actual log dir?  Opened https://github.com/msk-mind/backlog/issues/776